client/src/entrypoints/admin/page-editor.js

```diff
@@ -271,8 +275,96 @@ function initKeyboardShortcuts() {
     return false;
   });
 }
+
 window.initKeyboardShortcuts = initKeyboardShortcuts;
 
+function initHiddenBreadcrumbs() {
+  const breadcrumbsToggle = document.querySelector('[data-toggle-breadcrumbs]');
+  const breadcrumbLinks = [
+    ...document.querySelectorAll('[data-breadcrumb-link]'),
+  ];
```

Comment:
Our browser support means we can do the simpler:

```suggestion
  const breadcrumbLinks = document.querySelectorAll('[data-breadcrumb-link]');
```